src/inspector/main_thread_interface.h

```diff
@@ -0,0 +1,137 @@
+#ifndef SRC_INSPECTOR_MAIN_THREAD_INTERFACE_H_
+#define SRC_INSPECTOR_MAIN_THREAD_INTERFACE_H_
+
+#if !HAVE_INSPECTOR
+#error("This header can only be used when inspector is enabled")
+#endif
+
+#include "env.h"
+#include "inspector_agent.h"
+#include "node_mutex.h"
+
+#include <deque>
+#include <memory>
+#include <unordered_map>
+#include <unordered_set>
+
+namespace v8_inspector {
+class StringBuffer;
+class StringView;
+}  // namespace v8_inspector
+
+namespace node {
+namespace inspector {
+class Request {
+ public:
+  virtual void Call() = 0;
+  virtual ~Request() {}
+};
```

Comment:
Would it make sense to re-use `v8::Task` for this? It has the same structure and purpose.

Reply:
I think such reuse might be confusing, implying this is somehow V8 specific.